db/migrate/20230721120009_add_users_ref_to_likes.rb

```diff
@@ -0,0 +1,6 @@
+class AddUsersRefToLikes < ActiveRecord::Migration[7.0]
+    def change
+      add_reference :likes, :users, null: false, foreign_key: true
```

Comment:
- [ ] Kindly consider changing this migration because as in the EDR diagram, you should have `author_id` as foreign_key and not `user_id` in your Likes table. All you need to do is to have an `author_id` referencing User table instead of `user_id`.
What you can do is following:
first remove the foreign_key column
`rails generate migration RemoveForeignKeyAndIndexFromTableName`
then add following code in the new migration
```
def change
    # First, remove the foreign key constraint
    remove_foreign_key :child_table, :parent_table

    # Next, remove the index on the foreign key column
    remove_index :child_table, :parent_id

    # Finally, remove the foreign key column itself
    remove_column :child_table, :parent_id
  end
 ```
 then run `rails db:migrate`
 then run following command:
`rails generate migration ChangeUserRefToLikes;`
and add following code in the command
```
def change
    add_reference :likes, :author, foreign_key: { to_table: 'users'}
  end
``` 
 then run `rails db:migrate`